web/frontend/vite.config.ts

```diff
@@ -9,15 +7,6 @@ import url from 'node:url';
 const MD5 = new Hashes.MD5();
```

Comment:
I wonder with the new build system if we can reliably build hashes across envs now, may be a good follow-up here to avoid a potential issue in the future.

Reply:
Agreed! Do you remember what the exact issue was here? Would love to see if we can remove this.

Reply:
I am worried about this line: https://github.com/viamrobotics/rdk/pull/2522/files#diff-39089571ee7e571e992944f03599cc1597b49ad9c02742afbd13fbc25022bbbfR38

Reply:
Ugh github won't scroll to it, can you comment on the exact line?

Reply:
Ah I see, but that just happens on a CI runner and isn't explicitly committed? So shouldn't result in committed changes errors

Reply:
Cool, I'll investigate removing in the next follow up. There's more cleanup I need to do anyway that I'm saving for another PR since this was getting too big. (specifically, more things like `Client` and `statusStream` need to be moved into svelte stores to make them much easier to work with for new obstacle avoidance features)